hack/get-restic-ppc64le.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# Copyright 2019 the Velero contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [ -z "${RESTIC_VERSION}" ]; then
+    echo "RESTIC_VERSION must be set"
+    exit 1
+fi
+
+wget --quiet https://oplab9.parqtec.unicamp.br/pub/ppc64el/restic/restic-${RESTIC_VERSION}
```

Comment:
Who maintains this restic build?

Reply:
Restic builds are maintained by my team and hosted on that link. Also, we are already in the process to get it supported on power with Restic community.

Reply:
Yes our plan is to switch to GH release once it's up
CC - @rpsene